inbm/cloudadapter-agent/cloudadapter/cloud/client/inbs_cloud_client.py

```diff
@@ -141,14 +141,15 @@ def publish_update(self, key: str, value: str) -> None:
         timestamp = Timestamp()
```

Comment:
couple issues with `publish_update`

1. doc string refers to a `message` parameter but parameters are `key` and `value`
2. `message_dict = json.loads(value)` is done but there is no check against JSON schema